### PR TITLE
Main

### PR DESCRIPTION
**Pull request by RhysMcW**
Prog{x} Time select list kellerza#246

### CODE REVIEW

File: src/ha_addon_sunsynk_multi/a_sensor.py [hunk 1]
```diff
@@ -189,7 +189,7 @@ def on_change(val: float | int | str | bool) -> None:
             ent["icon"] = "mdi:clock"
             self.entity = SelectEntity(
                 **ent,
-                options=sensor.available_values(15, ist.get_state),
+                options=sensor.available_values(OPT.number_entity_mode, ist.get_state),
```
Comment (kellerza) [inline src/ha_addon_sunsynk_multi/a_sensor.py:192]: Should this not be `OPT.prog_time_interval`?

**Review by kellerza — COMMENTED** [review]
Thanks!

File: src/ha_addon_sunsynk_multi/a_sensor.py (hunk 1 above)
Comment (RhysMcW) [reply src/ha_addon_sunsynk_multi/a_sensor.py:192]: yep, you are correct

**Comment by kellerza** [comment]
Thanks @RhysMcW !

I made the ocnfig key optional & shortened the list. Hopefully everyone can update without any missing config keys

**Pull request merged**